index.html

```diff
@@ -30,7 +30,7 @@
             <img src="img/gamer 1.png" alt="" />
           </div>
           <div>
-            <h1>GAMERS</h1>
+            <h1>GAMERS  </h1>
```

Comment:
Logo is not like the design.
I already told you to take it as SVG or PNG, either do this or fix it to be like the design

Reply:
Done